compliance/aws/untagged_resources/aws_untagged_resources_meta_parent.pt

```diff
@@ -0,0 +1,970 @@
+
+## Meta Parent Parameters
+## These are params specific to the meta parent policy.
+parameter "param_combined_incident_email" do
+  type "list"
+  label "Email addresses for combined incident"
+  description "A list of email addresses to notify with the consolidated child policy incident."
+  default []
+end
+
+parameter "param_dimension_filter_includes" do
+  type "list"
+  label "Dimension Include Filters"
+  description <<-EOS
+  Filters [`dimension_name=dimension_value` pairs] to determine which AWS Accounts returned by the Flexera Bill Analysis API to **INCLUDE** and be applied to.
+  During each run this policy will select AWS Accounts who match **all** the filters defined and apply a child policy for each.
+  If no include filters are provided, then all AWS Accounts are included by default.
+  Most of the dimensions in Flexera can be used [default dimensions, custom tag dimensions, rule-based dimensions].  Full list of available dimensions documented in the [Bill Analysis API Docs](https://reference.rightscale.com/bill_analysis/).
+  EOS
+end
+
+parameter "param_dimension_filter_excludes" do
+  type "list"
+  label "Dimension Exclude Filters"
+  description <<-EOS
+  Filters [`dimension_name=dimension_value` pairs] to determine which AWS Accounts returned by  the Flexera Bill Analysis API to **EXCLUDE** and *not* have policy applied to.
+  During each run this policy will select AWS Accounts who match **all** the filters defined here and excludes them from results.
+  Can be used to exclude specific AWS Accounts [`vendor_account=123456789012`]
+  Most of the dimensions in Flexera can be used [default dimensions, custom tag dimensions, rule-based dimensions].  Full list of available dimensions documented in the [Bill Analysis API Docs](https://reference.rightscale.com/bill_analysis/).
+  EOS
+end
+
+parameter "param_policy_schedule" do
+  type "string"
+  label "Child Policy Schedule"
+  description "The interval at which the child policy checks for conditions and generates incidents."
+  default "daily"
+  allowed_values "daily", "weekly", "monthly"
+end
+
+## Child Policy Parameters
+parameter "param_allowed_regions_allow_or_deny" do
+  type "string"
+  label "Allow/Deny Regions"
+  description "Allow or Deny entered regions. See the README for more details"
+  allowed_values "Allow", "Deny"
+  default "Allow"
+end
+
+parameter "param_allowed_regions" do
+  type "list"
+  label "Regions"
+  allowed_pattern /^([a-zA-Z-_]+-[a-zA-Z0-9-_]+-[0-9-_]+,*|)+$/
+  description "A list of allowed or denied regions. See the README for more details"
+end
+
+parameter "param_tags_to_check" do
+  type "list"
+  label "Tag Keys"
+  category "AWS"
+  allowed_pattern /\S/
+  description "List of tag keys e.g., 'Environment' to find resources which are not tagged by the given inputs."
+  constraint_description <<-EOS
+    This is a mandatory field.
+  EOS
+end
+
+parameter "param_tags_to_add" do
```

Comment:
@bryankaraffa - might need some help on this one as this policy has remediation actions

I can see that this particular parameter `"param_tags_to_add"` is being taken from the parameter in the escalation `"esc_add_tags"` (Line 280 in the updated child policy), but it probably shouldn't be doing that I guess?

Reply:
Will cover this fix in a separate change.